jac-uk/admin
# #1864 QT platform integration

## What's included?
Our QT functionality has moved to the new QT platform. Here we are integrating with the new platform as well as streamlining/automating the processing of scores.

 - [x] Integrate with new QT platform
 - [x] Enable SLs to switch back from new platform to existing/old QTs
 - [x] Transfer candidate & reasonable adjustment data to new QT platform
 - [x] Import scores from new QT platform
 - [x] Process scores into ranked 'zones'
 - [x] Incorporate diversity stats
 - [x] Set a pass mark
 - [x] Override pass/fail status for individual candidates (within a zone)
 - [x] Support full screen mode
 - [x] Update status of applications based on pass/fail outcome
 - [x] Completion: lock things down & display key stats

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
For an exercise configured with Critical Analysis Test **and** Situational Judgement Test shortlisting methods please check that you can:
 - See three new links in the Tasks side menu: 
   - Critical Analysis Test
   - Situational Judgement Test
   - CA + SJ Scoring
 - Create a CA (and SJ) test
 - Transfer participants to the test, including reasonable adjustment data
 - [carry out the test on the new QT Platform]
 - Import scores back from the test
 - View ranked scores
 - Enter and exit full screen mode
 - Set an overall pass mark
 - Override individuals outcome
 - Mark as Completed
 - Check that applications status has been correctly updated

When you have completed both CA and SJ tests and scoring please go to 'CA + SJ scoring' and check that you can:
 - View ranked combined scores
 - Enter and exit full screen mode
 - Set an overall pass mark
 - Override individuals outcome
 - Mark as Completed
 - Check that applications status has been correctly updated

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
